### PR TITLE
chore(bigtable/bttest): remove half-baked instances support

### PR DESCRIPTION
**Pull request by dragonsinth**
A couple of years ago, support for DeleteInstance was added. However, this never really went anywhere; there's no way to actually create emulated instances or manage them.  Can we remove this cruft, and simplify unimplemented methods via modern protoc generated stubs?

### CODE REVIEW

**Comment by crwilcox** [comment]
@dragonsinth could you explain the problem this is solving? How does this change the user experience?

And thanks for your recent contributions to bttest. They are appreciated, but I want to make sure I understand your goal here :)

**Comment by dragonsinth** [comment]
This is just code hygiene, pure and simple!  It solves no particular problem, just removes some dead code and test to make maintenance easier going forward.

This is assuming no one has any plans to implement sophisticated instance management on the emulator.  Since this was added in 2019 figured it was probably a dead effort.

**Comment by crwilcox** [comment]
@igorbernstein2 any idea if this was something on the backlog?

File: bigtable/bttest/instance_server.go [hunk 1]
```diff
@@ -16,118 +16,26 @@ package bttest
...
-}
-
-func (s *server) SetIamPolicy(ctx context.Context, req *iampb.SetIamPolicyRequest) (*iampb.Policy, error) {
-	return nil, errUnimplemented
-}
-
-func (s *server) TestIamPermissions(ctx context.Context, req *iampb.TestIamPermissionsRequest) (*iampb.TestIamPermissionsResponse, error) {
-	return nil, errUnimplemented
+func (s *server) TestIamPermissions(_ context.Context, _ *iampb.TestIamPermissionsRequest) (*iampb.TestIamPermissionsResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method TestIamPermissions not implemented")
```
Comment (jimfulton) [inline bigtable/bttest/instance_server.go:40]: Why not move these placeholders to inmem.go and do away with this file entirely.  These are really just completing (for some definition of completing) BigtableInstanceAdminServer.

**Comment by jimfulton** [comment]
FWIW, I endorse removing dead code. Every line of code is a liability that needs to earn it's keep.  Realzy code can be added later if someone decides to implement instances.

File: bigtable/bttest/instance_server.go (hunk 1 above)
Comment (dragonsinth) [reply bigtable/bttest/instance_server.go:40]: 🤷  that file seems pretty cluttered already

**Comment by crwilcox** [comment]
While going over the backlog of PRs I reviewed this again. Were these by chance here to satisfy an interface?

**Comment by dragonsinth** [comment]
Perhaps-- but the interface isn't exposed over grpc so I'm not sure what the utility of it would be.  At least in OSS.  There might be Google internal refs

**Comment by codyoss** [comment]
@googleapis/api-bigtable Let's decide if we want to move forward with this or close out the issue.

**Pull request closed without merging**